### PR TITLE
Some fixes for takeover case

### PR DESCRIPTION
**Pull request by guihecheng**
This change is [Reviewable](https://reviewable.io/reviews/linux-ha-storage/storhaug/8)

### CODE REVIEW

File: src/storhaug [hunk 1]
```diff
@@ -301,10 +301,11 @@ create_virt_ip()
     pcs -f ${cibfile} constraint location vip${ipcount} rule resource-discovery=exclusive score=0 role eq storage
 
     pcs -f ${cibfile} resource create vip${ipcount}_trigger ocf:heartbeat:ganesha_trigger \
-        meta resource-stickiness="0"
+        --clone vip${ipcount}_trigger-clone vip${ipcount}_trigger \
+	meta interleave="true" globally-unique="false" notify="true" clone-max="${STORAGE_NUM_SERVERS}"
```
Comment (jarrpa) [inline src/storhaug]: I think I understand most of what you're doing, but why is `clone-max` not 1? Shouldn't there ever only be one, the one that follows the `vip${ipcount}` resource?

Comment (guihecheng) [reply src/storhaug]: @jarrpa oh, I think your are right, here should be 1.

**Comment by guihecheng** [comment]
@jarrpa updated clone-max=1, any more suggestions or comments?

**Comment by jarrpa** [comment]
LGTM!

**Pull request merged**